src/style.css

```
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+}
+
+body {
+  box-sizing: border-box;
+}
```

only this rule is allowed to use at * selector, so you can do it)
```suggestion
* {
  box-sizing: border-box;
}
```